.github/workflows/daily-test.yml

```diff
@@ -0,0 +1,41 @@
+name: Test template (daily)
+
+on:
+  workflow_dispatch: # allows manual triggering
+  schedule:
+    - cron: "8 0 * * *" # run on a daily basis
```

Comment:
Shouldn't it also run on PRs? Or at least on PRs that touch `flake.*` or `template`?

Reply:
We'll add that later. Since in PRs we need to actually check template in that PR.

Reply:
Oh yes, we have that good ol' chicken-and-egg problem :upside_down_face: